### DISCUSSION TITLE
Make "count of matches" optional for search and searchset sub-commands

### DISCUSSION THREAD

**Discussion by slonyik** [dathere/qsv#524] [Q&A] [answered]
Hello Joel,

The `qsv search` currently shows the found lines _and_ count of matches.
The output is then quite confusing.
Moreover piping multiple search `cat file | qsv search -s col1 pat1| qsv search -s col2 pat2` commands  gives  additional match/count numbers. 
Redirecting the stderr is not really practical :/

👉 I would like to propose the `qsv search` to not to print the number of matches by default,
and to add an optional parameter ( -c --count ) to search command to have the feature available.

PS : If we need to have count (after series of searches) we'd process the resulting rows  with `qsv count` 
PS: Looks like the qsv 0.62 did not have this feature.

Thanks in advance
BR
Miro

**Comment by jqnatividad** [comment, marked as answer]
Here you go @slonyik !

https://github.com/jqnatividad/qsv/pull/526

Lemme know if it works as expected...

**Reply by slonyik** [reply]
Works like a charm ! :)
Thank You !
